addons/self-referencing/addon.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Self referencing",
+  "description": "Adds the \"myself\" option to certain block dropdowns in the sensing category. Useful when working with clones.",
```

Comment:
To avoid confusing the user, we should find a different way to word "the myself option". Perhaps something like "Adds the option to get information from the selected sprite (in addition to other sprites) for certain blocks."